graderx/graders/unittest_graders/pytest/pytest_grader.py

```diff
@@ -0,0 +1,96 @@
+from pathlib import Path
+import shlex
+import subprocess
+import os
+from .lib.submissions_extraction import extract_submissions, clean_directory
+
+
+def get_course_root(course):
+    return Path(__file__).joinpath(
+        f'../../../courses/{course}/app/').resolve()
+
+
+def get_lab_path(course, lab):
+    return get_course_root(course).joinpath(lab)
+
+
+def run_grader(course, lab):
+    """
+    Runs the tests in the given lab's corresponding pytest test file 
+    which is test_run_grader.py in each lab directory  
+    then saves the output in a file
+    then pipes the pytest output file to console_log_parser 
+    which will parse the pytest output to create 4 files
+    1- [TIMESTAMP]_[LABNAME]_crash_details.txt which contains 
+    detailed pytest output of only the failed tests for all the submissions
+    2- [TIMESTAMP]_[LABNAME]_grade_summary.csv which contains
```

Comment:
is grade_summary.csv related (somehow) to [lab]_result_summary.txt (compute_results)?
shall compute_results return [lab]_result_summary.csv instead?